### PR TITLE
add scalacheck-1-14 → scalacheck-1-15 migration

### PR DESCRIPTION
**Pull request by larsrh**
This is the first artifactId migration in the config file, so I'm not 100% sure this syntax is correct.

### CODE REVIEW

File: modules/core/src/main/resources/artifact-migrations.conf [hunk 1]
```diff
@@ -22,5 +22,11 @@ changes = [
     groupIdAfter = org.typelevel
     artifactIdAfter = kind-projector
     initialVersion = 0.10.0
+  },
+  {
+    groupIdBefore = org.scalatestplus
```
Comment (mzuehlke) [inline modules/core/src/main/resources/artifact-migrations.conf:27]:
```suggestion
    groupIdAfter = org.scalatestplus
```
It's the other way around: https://github.com/scala-steward-org/scala-steward/blob/master/docs/artifact-migrations.md
The `*After` fields are mandantory.

**Review by mzuehlke — CHANGES_REQUESTED** [review]

**Review by mzuehlke — APPROVED** [review]

**Comment by mzuehlke** [comment]
thanks @larsrh

**Comment by larsrh** [comment]
This does not appear to trigger, e.g. in Paiges: https://github.com/typelevel/paiges

**Comment by fthomas** [comment]
Here is an excerpt from the log for Paiges:
```
2020-11-09 19:56:10,956 INFO  Ignore org.scalatestplus:(scalacheck-1-14, scalacheck-1-14_2.12) : 3.2.2.0 -> 3.3.0.0-SNAP2 (reason: no suitable next version)
2020-11-09 19:56:10,956 INFO  Ignore org.scalatestplus:(scalacheck-1-14, scalacheck-1-14_2.13) : 3.2.2.0 -> 3.3.0.0-SNAP2 (reason: no suitable next version)
2020-11-09 19:56:10,956 INFO  Ignore org.scalatestplus:(scalacheck-1-14, scalacheck-1-14_sjs1_2.12) : 3.2.2.0 -> 3.3.0.0-SNAP2 (reason: no suitable next version)
2020-11-09 19:56:10,956 INFO  Ignore org.scalatestplus:(scalacheck-1-14, scalacheck-1-14_sjs1_2.13) : 3.2.2.0 -> 3.3.0.0-SNAP2 (reason: no suitable next version)
```
The migration added here is not used because there are newer versions of that artifact available. Scala Steward currently only rewrites groupIds or artifactIds if there are no newer versions for the given artifact.

**Comment by larsrh** [comment]
In that case, I think we're out of luck, since upstream will probably continue to release new versions.

**Pull request merged**